manager/ui/war/plugins/api-manager/ts/consumer/consumer-orgs.ts

```diff
@@ -19,6 +22,8 @@ module Apiman {
                         var body:any = {};
                         body.filters = [];
                         body.filters.push( {"name": "name", "value": "*" + params.q + "*", "operator": "like"});
+                        body.page = 1;          //default on backend for some functions
+                        body.pageSize = 500;    //default on backend for some functions
```

Comment:
@tevosouza As in my [comment](https://github.com/apiman/apiman/pull/806#issuecomment-584013482) here I think you should set the pageSize from the dropdown. Otherwise, you would only get 500 entries in a wildcard search :)

Reply:
Hi,
This was really my approach, even though it was already implemented in the backend, I used 2 criteria to decide

1- In a wildcard search, in theory, the user knows what he is looking for if by chance he does a search using '*' I believe that limiting the number of records returned is not a problem.
2- Because it is a search screen, I found it less traumatic to make just one call to bring the results and page only on the front end.

I believe that because they are organizations, there is hardly a use case (in my opinion) that the user needs to do a search that returns more than 500 lines and uses all this data.

Reply:
@tevosouza Maybe you can spend just a little more effort to compute the requested page size from the backend :-)
Like @volkflo said, use the pageSize from the user and calculate in the front end the requested page number. You just need a check if the current returned result from the backend allows a click on the next/previous button :-)

for the next button:
`totalSize < pageSize * page`
or more concretely:
`totalSize < (lastPageNumber * pageSize) + currentResultCount`

Reply:
@tevosouza You are almost there :-)

Reply:
@bekihm and @volkflo, you convinced me, I'll change my approach of pagination and make a new pull-request later, by now I'll close this pull-request.1

Thanks for your comments!!